node/network/collator-protocol/src/collator_side/mod.rs

```diff
@@ -987,7 +991,9 @@ where
 	let validators: Vec<_> = group_validators.into_iter().collect();
 	let no_one_is_assigned = validators.is_empty();
 	if no_one_is_assigned {
-		tracing::warn!(target: LOG_TARGET, "No validators assigned to our core.",);
+		if let Some(core) = maybe_core {
+			tracing::warn!(target: LOG_TARGET, ?core, "No validators assigned to our core.",);
+		}
```

Comment:
Shouldn't we log something like `A core is occupied` in the `else` branch? As far as I see `determine_core` returns `None` in this case.

Reply:
This is expected every second block until we have async backing. I'll add a debug log as it doesn't hurt.